raiden/api/python.py

```diff
@@ -228,6 +228,13 @@ def channel_open(
             raise DuplicatedChannelError('Channel with given partner address already exists')
 
         registry = self.raiden.chain.token_network_registry(registry_address)
+        token_network_address = registry.get_token_network(token_address)
+
+        if token_network_address is None:
+            raise InvalidAddress(
```

Comment:
Nice fine Jozef! Can you do a small change though?

Create a new exception, say 'TokenNotRegistered` with an appropriate error like:` "Token %s is not registered. Please first call register token before any channel operation"`.

And then handle that exception [here](https://github.com/raiden-network/raiden/blob/master/raiden/api/rest.py#L399) where the other 409s are returned.

Reply:
done!